models/Translation.php

```diff
@@ -200,6 +215,38 @@ public function setDomain(string $domain): void
         $this->domain = !empty($domain) ? $domain : self::DOMAIN_DEFAULT;
     }
 
+    /**
+     * @return int|null
+     */
+    public function getUserOwner(): ?int
+    {
+        return $this->userOwner;
+    }
+
+    /**
+     * @param int|null $userOwner
+     */
+    public function setUserOwner(?int $userOwner)
```

Comment:
```suggestion
    public function setUserOwner(?int $userOwner): void
```

Reply:
we don't add `void` return type-hints for simplicity.

Reply:
What? :confused:

Reply:
just to be consistent with other models as we avoid the overhead of adding `void` return type to the existing methods in models.

Reply:
@dvesh3 Than we can't get a higher PhpStan level. The next level 6 want to set return types. When there is no void you have to set "@return void" in the doc

Reply:
Ok, that's a good point. @fashxp @brusch I think we to consider this change for our models. wdyt?

Reply:
@dvesh3 yep, that's a valid point, better having proper ` : void` than `@return void` imho.

Reply:
We decided to add `:void` return typehints in new methods and then we can add `@return void` phpdoc to models on Pimcore 11 to upgrade phpstan level 6.